### DISCUSSION TITLE
Can't change repository URL

### DISCUSSION THREAD

**Discussion by AngryVelociraptor** [codemagic-ci-cd/codemagic-docs#1981] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I would like to change my repository URL. I can't do so.

wanted URL: https://github.com/exbrain-app/exbrain
app URL: https://codemagic.io/app/64068fe5c261218545613d82

See previous discussion.
https://github.com/orgs/codemagic-ci-cd/discussions/1920
I hope this is a bug you're tracking. It happened after I deleted the previous repository

### Expected results

Repository URL is editable

### Actual results

Repository URL is readonly

**Comment by icarusdust** [comment]
Hi @AngryVelociraptor, we make these changes from our end, you are not expected to make these changes yourself, therefore it is not a bug but an expected behaviour. Please provide the details that @kulakarla asked in the previous thread, so that we can make these changes for you.
